mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_backup_plan_association_test.go.tmpl

```diff
@@ -0,0 +1,172 @@
+package backupdr_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+	"fmt"
+	"strings"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+)
+
+func TestAccDataSourceGoogleBackupDRBackupPlanAssociation_basic(t *testing.T) {
```

Comment:
[The test failure](https://github.com/GoogleCloudPlatform/magic-modules/pull/12159#issuecomment-2440624535) here is: 

```
        Error: Invalid resource type
        
          on terraform_plugin_test.tf line 55, in resource "google_backup_dr_backup_plan" "foo":
          55: resource "google_backup_dr_backup_plan" "foo" { 
        
        The provider hashicorp/google does not support resource type
        "google_backup_dr_backup_plan".
```

When I look at the `provider_mmv1_resources.go` file that is [generated in the google-beta provider from this PR](https://github.com/modular-magician/terraform-provider-google-beta/blob/auto-pr-12159/google-beta/provider/provider_mmv1_resources.go) I cannot see `google_backup_dr_backup_plan` listed there. Have you already had a PR accepted for adding that resource? If so you may need to pull those changes into your branch.